docs/features/configuring_workflows.md

```diff
@@ -0,0 +1,21 @@
+# Configuring Your Workflows
+
+DAG Factory allows you to define workflows in a structured, configuration-driven way using YAML files.
```

Comment:
It may be worth mentioning that one YAML file can contain multiple workflows, not sure if we're saying this somewherelse.

Reply:
yes, added such a line here, thanks.